src/components/network/config-tables.js

```diff
@@ -711,12 +729,26 @@ export const TABLES_DEFINITIONS = {
                 dataKey: 'p0',
                 numeric: true,
                 fractionDigits: 1,
+                changeCmd: 'equipment.setP0({})\n',
+                editor: ({ equipment, ...props }) =>
```

Comment:
undefined props don't need to be specified, I think you can simplify this to :
editor: NumericalField,

Reply:
yes, right. I will simplify as requested